.github/CODEOWNERS

```diff
@@ -1,110 +1,73 @@
-
-# Half Type
-sycl/include/CL/sycl/half_type.hpp @AlexeySachkov
-sycl/source/half_type.cpp @AlexeySachkov
-
 # XPTI instrumentation utilities
-xpti/ @tovinkere @andykaylor @alexbatashev
-xptifw/ @tovinkere @andykaylor @alexbatashev
+xpti/ @tovinkere @intel/llvm-reviewers-runtime
+xptifw/ @tovinkere @intel/llvm-reviewers-runtime
 
 # DPC++ tools
-llvm/**/append-file/ @mdtoguchi @AGindinson
-llvm/**/file-table-tform/ @mlychkov @AlexeySachkov @kbobrovs
-llvm/**/llvm-foreach/ @AlexeySachkov  @Fznamznon
-llvm/**/llvm-no-spir-kernel/ @AGindinson @AlexeySachkov
-llvm/**/sycl-post-link/ @mlychkov @AlexeySachkov @kbobrovs
-llvm/include/llvm/Support/PropertySetIO.h @mlychkov @AlexeySachkov @kbobrovs
-llvm/lib/Support/PropertySetIO.cpp @mlychkov @AlexeySachkov @kbobrovs
-llvm/unittests/Support/PropertySetIOTest.cpp @mlychkov @AlexeySachkov @kbobrovs
-llvm/include/llvm/Support/SimpleTable.h @mlychkov @AlexeySachkov @kbobrovs
-llvm/lib/Support/SimpleTable.cpp @mlychkov @AlexeySachkov @kbobrovs
-llvm/unittests/Support/SimpleTableTest.cpp @mlychkov @AlexeySachkov @kbobrovs
-llvm/lib/Support/Base64.cpp @mlychkov @AlexeySachkov @kbobrovs
+llvm/**/append-file/ @intel/dpcpp-tools-reviewers
+llvm/**/file-table-tform/ @intel/dpcpp-tools-reviewers
+llvm/**/llvm-foreach/ @intel/dpcpp-tools-reviewers
+llvm/**/llvm-no-spir-kernel/ @intel/dpcpp-tools-reviewers
+llvm/**/sycl-post-link/ @intel/dpcpp-tools-reviewers
+llvm/include/llvm/Support/PropertySetIO.h @intel/dpcpp-tools-reviewers
+llvm/lib/Support/PropertySetIO.cpp @intel/dpcpp-tools-reviewers
+llvm/unittests/Support/PropertySetIOTest.cpp @intel/dpcpp-tools-reviewers
+llvm/include/llvm/Support/SimpleTable.h @intel/dpcpp-tools-reviewers
+llvm/lib/Support/SimpleTable.cpp @intel/dpcpp-tools-reviewers
+llvm/unittests/Support/SimpleTableTest.cpp @intel/dpcpp-tools-reviewers
+llvm/lib/Support/Base64.cpp @intel/dpcpp-tools-reviewers
 
 # Clang offload tools
-clang/tools/clang-offload-bundler/ @mlychkov @sndmitriev @AlexeySachkov
-clang/tools/clang-offload-wrapper/ @mlychkov @sndmitriev @AlexeySachkov @kbobrovs
-clang/tools/clang-offload-deps/ @sndmitriev @mlychkov @AlexeySachkov
-clang/tools/clang-offload-extract/ @sndmitriev @mlychkov @AlexeySachkov
+clang/tools/clang-offload-bundler/ @intel/dpcpp-tools-reviewers
+clang/tools/clang-offload-wrapper/ @intel/dpcpp-tools-reviewers
+clang/tools/clang-offload-deps/ @intel/dpcpp-tools-reviewers
+clang/tools/clang-offload-extract/ @intel/dpcpp-tools-reviewers
 
 # Explicit SIMD
-SYCLLowerIR/ @kbobrovs @sndmitriev @kychendev @v-klochkov 
-esimd/ @kbobrovs @v-klochkov @kychendev
-sycl/include/sycl/ext/intel/experimental/esimd.hpp @kbobrovs @v-klochkov @kychendev
-sycl/doc/extensions/ExplicitSIMD/ @kbobrovs @v-klochkov @kychendev
+SYCLLowerIR/ @intel/dpcpp-esimd-reviewers
+esimd/ @kbobrovs @intel/dpcpp-esimd-reviewers
+sycl/include/sycl/ext/intel/experimental/esimd.hpp @intel/dpcpp-esimd-reviewers
+sycl/doc/extensions/ExplicitSIMD/ @intel/dpcpp-esimd-reviewers
 
 # ITT annotations
```

Comment:
We can remove this section if @intel/dpcpp-tools-reviewers team okay to review changes in llvm/ directory.